.env.example

```
@@ -1,9 +1,8 @@
-ETHERSCAN_API_KEY=
-RPC_URL=
-PRIVATE_KEY=
+MNEMONIC_INDEX=
+LEDGER_SENDER=
 
 # Test rpc_endpoints
-RPC_ETHEREUM=https://rpc.flashbots.net
+RPC_MAINNET=https://rpc.flashbots.net
 RPC_AVALANCHE=https://api.avax.network/ext/bc/C/rpc
 RPC_OPTIMISM=https://mainnet.optimism.io
```

```suggestion
RPC_OPTIMISM=https://endpoints.omniatech.io/v1/op/mainnet/public
```

seems this rpc has less issues with old blocks :man_shrugging: